packages/graphics/lcms2/package.mk

```
@@ -2,11 +2,11 @@
 # Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="lcms2"
-PKG_VERSION="2.12"
-PKG_SHA256="18663985e864100455ac3e507625c438c3710354d85e5cbb7cd4043e11fe10f5"
+PKG_VERSION="2.13.1"
+PKG_SHA256="d473e796e7b27c5af01bd6d1552d42b45b43457e7182ce9903f38bb748203b88"
 PKG_LICENSE="MIT/GPLv3"
 PKG_SITE="http://www.littlecms.com"
-PKG_URL="${SOURCEFORGE_SRC}/lcms/lcms/${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
+PKG_URL="${SOURCEFORGE_SRC}/lcms/lcms/$(get_pkg_version_maj_min)/${PKG_NAME}-${PKG_VERSION}.tar.gz"
```

```suggestion
PKG_URL="https://github.com/mm2/Little-CMS/releases/download/lcms${PKG_VERSION}/lcms2-${PKG_VERSION}.tar.gz"
```
untested, one less SF crap :)

---

tested